packages/product-json-to-xlsx/src/cli.js

```diff
@@ -201,6 +206,7 @@ Promise.all([getHeaders(args), resolveCredentials(args)])
       fillAllRows: args.fillAllRows,
       language: args.language,
       multiValueDelimiter: args.multiValueDelimiter,
+      onlyMasterVariants: args.onlyMasterVariants,
```

Comment:
Maybe we should also add this here? https://github.com/commercetools/nodejs/blob/c7daa22c81415497cfc8041360ade51e4dd1d58a/packages/product-json-to-xlsx/test/main.spec.js#L19

Reply:
It has a default value `onlyMasterVariants = false` so it worked even without it.
I added it to be more clear + it wanted also to bump flow version 0d744ef